app.py

```diff
@@ -143,7 +143,7 @@ def users_show(user_id):
     """Show user profile."""
 
     user = User.query.get_or_404(user_id)
-
+    likes =Likes.query.filter_by(user_id=user.id).count()
```

Comment:
You have a likes relationship on the user class so should be able to do `len(user.likes)` here